packages/components/src/components/page-overlay/page-overlay.scss

```diff
@@ -19,7 +19,7 @@
     }
     &--enter-done {
         transform: translateY(0);
-        opacity: 1;
+        opacity: 1 !important;
```

Comment:
Why is adding `!important` the fix? The `&--enter-done` lifecycle class should be already scoped enough not to have other class rules override this.

Reply:
@zakhov just sorted it.